docs/src/devdocs/interpolations.md

```diff
@@ -1,12 +1,51 @@
+```@meta
+Draft = false
+```
+
 # [Interpolations](@id devdocs-interpolations)
 
-## Type definitions
+All `Interpolation`s should subtype `Interpolation{shape, order}`,
+where `shape <: AbstractRefShape` is the reference shape for which
+the interpolation is defined and `order` is the characteristic interpolation
+order. The [how-to at bottom of this page](@ref devdocs-howto_new-interpolation) describes how to implement a new interpolation.
+
+## Methods to be implemented for a new interpolation
+### Always
+```@docs
+Ferrite.reference_shape_value(::Interpolation, ::Vec, ::Int)
+Ferrite.reference_coordinates(::Interpolation)
+Ferrite.vertexdof_indices(::Interpolation)
+Ferrite.facedof_indices(::Interpolation)
+Ferrite.facedof_interior_indices(::Interpolation)
+Ferrite.edgedof_indices(::Interpolation)
+Ferrite.edgedof_interior_indices(::Interpolation)
+Ferrite.volumedof_interior_indices(::Interpolation)
+Ferrite.getnbasefunctions(::Interpolation)
+Ferrite.adjust_dofs_during_distribution(::Interpolation)
+```
 
-Interpolations are subtypes of `Interpolation{shape, order}`, i.e. they are
-parametrized by the reference element and its characteristic order.
+### For special interpolations
+#### Discontinuous interpolations
+For discontinuous interpolations, the following methods should
+be implemented.
+```@docs
+Ferrite.is_discontinuous(::Interpolation)
+Ferrite.dirichlet_vertexdof_indices(::Interpolation)
+Ferrite.dirichlet_facedof_indices(::Interpolation)
+Ferrite.dirichlet_edgedof_indices(::Interpolation)
```

Comment:
Actually these are optional right now, as most "L2 continuous" elements have no dofs at their boundary.

Reply:
Not sure if I follow? The default is calling the non-dirichlet variant of these, but for L2 interpolations we do
https://github.com/Ferrite-FEM/Ferrite.jl/blob/1807f667086d12ec2b6a567264670980a309647c/src/interpolations.jl#L465-L467
to get dofs at the boundary?